apps/frontend/src/index.tsx

```diff
@@ -24,7 +25,7 @@ createRoot(document.querySelector("#root") as HTMLElement).render(
 							{
 								element: (
 									<ProtectedRoute
-										component="Root"
+										component={<QuizCategoriesForm submitLabel="Retake Quiz" />}
```

Comment:
I don't think we need this label here. Also you should display your component in /chat route